internal/host/hostcommands/tang_connectivity_check_cmd.go

```diff
@@ -26,14 +27,27 @@ func NewTangConnectivityCheckCmd(log logrus.FieldLogger, db *gorm.DB, tangConnec
 	}
 }
 
-func (c *tangConnectivityCheckCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models.Step, error) {
-	var cluster common.Cluster
-
-	if err := c.db.First(&cluster, "id = ?", host.ClusterID).Error; err != nil {
-		c.log.WithError(err).Errorf("failed to fetch cluster %s", host.ClusterID)
-		return nil, err
+func (c *tangConnectivityCheckCmd) getTangServersFromHostIgnition(host *models.Host) ([]byte, error) {
+	var (
+		luks        *ignition_types.Luks
+		tangServers []byte
+		err         error
+	)
+	if host.APIVipConnectivity != "" {
+		luks, err = hostutil.GetDiskEncryptionForDay2(c.log, host)
+		if err != nil {
+			return nil, err
+		}
+		if len(luks.Clevis.Tang) != 0 {
+			if tangServers, err = json.Marshal(luks.Clevis.Tang[0]); err != nil {
+				return nil, err
+			}
+		}
 	}
+	return tangServers, nil
```

Comment:
That would return `nil, nil` on if API VIP connectivity test result is missing

Reply:
Done.